src/test/java/com/google/api/codegen/testdata/csharp_gapic_snippets_library.baseline

```diff
@@ -0,0 +1,783 @@
+============== file: Google/Example/Library/V1/LibraryServiceClientSnippets.g.cs ==============
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+using Google.Api.Gax;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+using UpdateBookIndexRequest.Types;
+
+namespace Google.Example.Library.V1.Snippets
+{
+    public class LibraryServiceClientSnippets
+    {
+        public async Task CreateShelfAsync()
+        {
+            // Snippet: CreateShelfAsync(Shelf,CallSettings)
+            // Additional: CreateShelfAsync(Shelf,CancellationToken)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            Shelf shelf = new Shelf();
+            // Make the request
+            Shelf response = await libraryServiceClient.CreateShelfAsync(shelf);
+            // End snippet
+        }
+
+        public void CreateShelf()
+        {
+            // Snippet: CreateShelf(Shelf,CallSettings)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            Shelf shelf = new Shelf();
+            // Make the request
+            Shelf response = libraryServiceClient.CreateShelf(shelf);
+            // End snippet
+        }
+
+        public async Task GetShelf1Async()
```

Comment:
It is very odd to see a number in the middle of an identifier like this instead of at the end...